digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -555,10 +551,10 @@ object Digiroad2Context {
   }
 
   lazy val revision: String = {
-    revisionInfo.getProperty("digiroad2.revision")
+    Digiroad2Properties.revision
```

Comment:
Mihin näitä käytettään? Riviltä 746 löytyi joku env haku. Pitää varmaan tarkistaa mitä sillä tehdään ja keskitetään tonne digiroad2properties jos tarvetta